Add ZeroReact.NET to Readme

DaniilSokolyuk opened the pull request.

Daniel15 commented:
@DaniilSokolyuk  Would you be interested in submitting some pull requests to ReactJS.NET to pull in your changes?

DaniilSokolyuk commented:
@Daniel15, hi, I think it is very difficult because i use **only** ChakraCore and modified version of JavaScriptEngineSwitcher.ChakraCore, there have been many changes, 
I changed the [ScriptDispatcher](https://github.com/DaniilSokolyuk/ZeroReact.NET/blob/95d21b47efe3c9c523fb95630af6f4e2d03b03c4/src/JavaScriptEngineSwitcher.ChakraCore/Ext/ScriptDispatcherExt.cs#L20) implementation, now he uses shared queue for reduce the cost of getting and returning the engine, a [new evaluate](https://github.com/DaniilSokolyuk/ZeroReact.NET/blob/95d21b47efe3c9c523fb95630af6f4e2d03b03c4/src/JavaScriptEngineSwitcher.ChakraCore/Ext/ChakraCoreJsEngineExt.cs#L13) method has been added, he accepts ReadOnlyMemory and return IMemoryOwner for 
My [JsPool](https://github.com/DaniilSokolyuk/ZeroReact.NET/blob/master/src/ZeroReact/JsPool/ZeroJsPool.cs) uses different threads to create and maintenange engines in the background not during the execution of the query, also ZeroJsPool and ChakraCore engine is very coupled
